refactor: refactor readme and app.py for readability

TsubasaK111 opened the pull request:
refactored README of both languages and app.py template to improve readability.

TsubasaK111 committed:
refactor: refactor readme and app.py for readability

refactored README of both languages and app.py template to improve readability.

TsubasaK111 commented:
@shunjikonishi review me!

shunjikonishi commented on app/app.py:
```diff
@@ -1,4 +1,5 @@
-  for v in args:
-    print(v)
+  for arg in args:
+    result = do_something(arg)
```
do_something is not defined.

shunjikonishi commented:
@TsubasaK111 fix me
app.py should work without editing.
As default, app.py works as an echo application.

TsubasaK111 committed:
fix: change app.py to successfully run by default

App.py should successfully run even with no code change. Thus, the default behavior is now a simple echo back.

TsubasaK111 commented:
@shunjikonishi As you requested, app.py now works without editing, and echoes back. Review me!

shunjikonishi commented:
![LGTM](http://lgtm.in/p/2Ht1N6hjm)
